plugin/evm/header/dynamic_fee_windower.go

```diff
@@ -20,29 +18,21 @@ import (
 	"github.com/ethereum/go-ethereum/common/math"
 )
 
-// FeeWindowSize is the number of bytes that are used to encode the dynamic fee
-// window in the header's Extra field after the Apricot Phase 3 upgrade.
-const FeeWindowSize = wrappers.LongLen * ap3.WindowLen
-
 var (
 	maxUint256Plus1 = new(big.Int).Lsh(common.Big1, 256)
 	maxUint256      = new(big.Int).Sub(maxUint256Plus1, common.Big1)
 
-	ap3MinBaseFee = big.NewInt(ap3.MinBaseFee)
-	ap4MinBaseFee = big.NewInt(ap4.MinBaseFee)
-	// EtnaMinBaseFee is exported so that it can be modified by tests.
-	//
-	// TODO: Unexport this.
-	EtnaMinBaseFee = big.NewInt(etna.MinBaseFee)
```

Comment:
This didn't need to be exported anymore because the tests that needed to use this now use the `acp176.MinBaseFee` - which is `1 wei`, which was what this was being overridden to.